src/Workspaces/Core/Portable/Rename/ConflictEngine/ConflictResolver.Session.cs

```diff
@@ -26,7 +26,7 @@ internal static partial class ConflictResolver
         /// Helper class to track the state necessary for finding/resolving conflicts in a 
         /// rename session.
         /// </summary>
-        private class Session
+        internal class Session
```

Comment:
why internal?

Reply:
I'm removing this. It doesn't help find the right errors up front. I'll file a separate work item. There's other code changes that would benefit from being able to see if rename conflicts would happen.